restapi/src/test/java/io/stargate/web/docsapi/service/DocumentServiceTest.java

```diff
@@ -495,122 +472,6 @@ void testGetDocPathFields() throws JsonProcessingException {
     return result;
   }
 
-  @Test
-  void testSearchDocEmpty() throws JsonProcessingException {
-    withQuery(
-            table,
-            "SELECT key, leaf FROM test_docs.collection1 WHERE p0 = ? AND p1 > ? AND p2 = ? AND p3 = ? AND dbl_value > ? ALLOW FILTERING",
-            params("a", "", "c", "", 1.0))
-        .returningNothing();
-    DocumentResponseWrapper<Map<String, ?>> r = searchDoc("{\"a.*.c\":{\"$gt\":1}}", null);
-    assertThat(r.getPageState()).isNull();
-    assertThat(r.getData()).isEqualTo(m());
-  }
-
-  @Test
-  void testSearchDoc() throws JsonProcessingException {
```

Comment:
Where do we keep equivalent tests now? Sorry, I could not find them :thinking:

Reply:
I wrote in the PR description that the unit tests are still missing..

Reply:
Oops, sorry, I saw that, but somehow assumed you added new tests already :)